src/components/Header/Header.module.scss

```diff
@@ -0,0 +1,164 @@
+.header {
+  &__content {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin: 0 260px;
+  }
+
+  &__add {
+    background: #52106a;
+    height: 42px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+    padding: 0 15px;
+
+    h4 {
```

Comment:
add class to not add styles to an empty tag

Reply:
https://github.com/igorizviekov/light-up/pull/7/commits/e1b8f39f5217f487124296a507c3e31c860d8306